resource/translations/messages.se.json

```diff
@@ -40,6 +40,7 @@
     "Error: Term \"%term%\" not found in vocabulary!": "Error: Tearbma \"%term%\" ii g\u00e1vdnon s\u00e1tner\u00e1jus!",
     "Feedback": "M\u00e1hcahat",
     "Feedback has been sent!": "M\u00e1hcahat lea s\u00e1ddejuvvon!",
+    "Go to the concept page": "KÄÄNNETTÄVÄ",
```

Comment:
Remove this - I think it's better to leave out translations that aren't known yet